setup.py

```diff
@@ -138,12 +138,12 @@ def run(self):
             if e.errno != errno.EEXIST:
                 raise
 
-        # Ensure all of our executanle files have their permission set
+        # Ensure all of our executable files have their permission set
         for filename in [
                 "src/libsodium/autogen.sh",
                 "src/libsodium/compile",
-                "src/libsodium/config.guess",
-                "src/libsodium/config.sub",
+                "src/libsodium/build-aux/config.guess",
+                "src/libsodium/build-aux/config.sub",
```

Comment:
By looking at how the commands involving `config.guess` and `config.sub` are invoked in ./configure:
```
`$SHELL "$ac_aux_dir/config.guess"`
`$SHELL "$ac_aux_dir/config.sub"`
```
it seems the execute permission is not needed anymore for the configure step to succeed.
I'd suggest to just remove these files from the list, and verify the library is still correctly built.

Reply:
After checking the boundled version is equal to the one extracted from a pristine upstream tar, I just verified the git worktree stays clean after building. Merging.